api/resources/light_curve/models.py

```diff
@@ -1,4 +1,33 @@
 from flask_restx import Resource, fields, Model
+from sqlalchemy import except_
+
+def get_magpsf(raw_response):
+    try:
+        magpsf = raw_response.magpsf
+        return magpsf
+    except:
+        pass
+    try:
+        mag = raw_response.get("mag")
+        return mag
+    except:
+        pass
+
```

Comment:
I would try to avoid `except:pass`

https://realpython.com/the-most-diabolical-python-antipattern/